README.md

```diff
@@ -737,6 +737,39 @@ for family in text_string_to_metric_families(u"my_gauge 1.0\n"):
     print("Name: {0} Labels: {1} Value: {2}".format(*sample))
 ```
 
+## Restricted registry
+
+Registry support querying a specific metric with the GET parameter "name[]".
+Since it's an array it can be use multiple times.
+
+```python
+curl -v --get --data-urlencode "name[]=python_gc_objects_collected_total" --data-urlencode "name[]=python_info" http://127.0.0.1:9200/metrics
+* About to connect() to 127.0.0.1 port 9200 (#0)
+*   Trying 127.0.0.1...
+* Connected to 127.0.0.1 (127.0.0.1) port 9200 (#0)
+> GET /metrics?name[]=python_gc_objects_collected_total&name[]=python_info HTTP/1.1
+> User-Agent: curl/7.29.0
+> Host: 127.0.0.1:9200
+> Accept: */*
+>
+* HTTP 1.0, assume close after body
+< HTTP/1.0 200 OK
+< Date: Thu, 19 Oct 2023 10:00:38 GMT
+< Server: WSGIServer/0.2 CPython/3.9.3
+< Content-Type: text/plain; version=0.0.4; charset=utf-8
+< Content-Length: 454
+<
```

Comment:
Having the verbose output is unnecessary in the readme.

Reply:
done